.github/workflows/wheels-test.sh

```diff
@@ -1,12 +1,20 @@
 #!/bin/bash
 set -e
 
+# Ensure fribidi is installed by the system.
 if [[ "$OSTYPE" == "darwin"* ]]; then
-    brew install fribidi
-    export PKG_CONFIG_PATH="/usr/local/opt/openblas/lib/pkgconfig"
-    if [ -f /opt/homebrew/lib/libfribidi.dylib ]; then
-        sudo cp /opt/homebrew/lib/libfribidi.dylib /usr/local/lib
+    # If Homebrew is on the path during the build, it may leak into the wheels.
+    # However, we *do* need Homebrew to provide a copy of fribidi for
+    # testing purposes so that we can verify the fribidi shim works as expected.
+    if [[ "$(uname -m)" == "x86_64" ]]; then
+        HOMEBREW_HOME=/usr/local/homebrew
```

Comment:
```suggestion
        HOMEBREW_HOME=/usr/local
```
I would have thought just '/usr/local', based on https://docs.brew.sh/Installation
> The script installs Homebrew to its default, supported, best prefix (/opt/homebrew for Apple Silicon, /usr/local for macOS Intel

Reply:
Turns out both work (for `brew`) at least - Homebrew keeps the "original" versions in /usr/local/homebrew. However, I agree that `/usr/local` is the generally intended entry point.

Reply:
I take that back - it *does* matter (and making this change caused test failures on x86_64).

Firstly, it's possible for homebrew to have packages *installed*, but not *linked*. Using the `/usr/lib/Homebrew` location ensures that the binary can be used regardless (note also the capitalisation fix - doesn't strictly matter on HFS+, but for accuracy).

Secondly, it's possible for *other* libraries to be in `/usr/local/lib`. Using `/usr/local/Homebrew/lib` for the `DYLD_LIBRARY_PATH` avoids this.